app/main/views.py

```diff
@@ -58,10 +57,54 @@ def get_contact_tables(firm: Firm, head_office: Office = None) -> list[DataTable
     return contact_tables
 
 
-def provider_name_html(provider: Firm):
-    return (
-        f"<a class='govuk-link', href={url_for('main.view_provider', firm=provider.firm_id)}>{provider.firm_name}</a>"
+def get_payment_information_table(firm: Firm, office: Office) -> DataTable:
+    table = SummaryList()
+    table.add_row(value=None, label="Payment method", row_action_urls={"enter": "#", "change": "#"})
+    return table
+
+
+def get_vat_registration_table(firm: Firm, office: Office) -> DataTable:
```

Comment:
I think this function already exists on the main branch

Reply:
agree, this table is on line 317 in main